core/ir/aarch64/instr_create_api.h

```diff
@@ -1693,6 +1693,56 @@ enum {
 #define INSTR_CREATE_bif_vector(dc, Rd, Rm, Rn) \
     instr_create_1dst_2src(dc, OP_bif, Rd, Rm, Rn)
 
+/**
+ * Creates a FCVTAS vector instruction.
```

Comment:
Hmm, I would pronounce it "eff-convert-az" and thus s/a/an/.  Are you saying it "fuh-convert-az"?  If so, leave it -- just curious.  Ditto below.

Reply:
This was annoying me as well. I left it as "a" instead of "an" because many things surrounding it were using "a" as well. I'll change it.

Reply:
Done